modules/products/bitbucket/nfs/nfs-server/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+NFS Server installed.
```

Comment:
What is this for?

Reply:
The `NOTES.txt` file is printed after the helm chart installation happens.